.github/workflows/workflow-release-services.yaml

```diff
@@ -48,11 +46,28 @@ jobs:
       - name: Build with Maven
         run: ./mvnw --batch-mode clean install -DskipTests
 
+      #
+      # The GPG key is stored in the secret using the text 'PGP Private Key' format
+      # Import it to GPG keyring and trust it.
+      #
+      - name: Import GPG key
+        if: inputs.maven-release == true
+        run: echo "${{ secrets.gpg_private_key }}" | gpg --import --no-tty --batch --yes --always-trust
+
+      #
+      # Show available keys. This will hide the key name (since it is in the secrets), but show for example expiration date
+      #
+      - name: List GPG keys available
+        if: inputs.maven-release == true
+        run: gpg --list-secret-keys --keyid-format=long
```

Comment:
```suggestion

```

@zambrovski Bracht man das list gpg keys noch? Das war doch nur zum debugging, oder?

Reply:
jain... die meldung des plugins finde ich extrem kryptisch... er sagt bei einem "expired" key - "failed to use unusable key". Der Begriff von "unusable key" kommt tatsächlich von GPG - z.B: sind expired keys unusable...

Mit der "list" - steht ganz klar wann der key expired. finde ich nicht verkehrt...